troubleshooting/app-does-not-connect-to-remote-point.md

```diff
@@ -0,0 +1,53 @@
+---
+title: App Cannot Connect to a Remote Service via HTTPS When Running on Device
+description: App cannot connect to a web service via HTTPS when deployed on a device and produces an unspecified error.
+page_title: App Cannot Connect to Remote Service via HTPS When Running on Device
+slug: app-cannot-connect to remote-service
+tags: connect, ios, web-service
+ticketid: 1061272, 1029681, 1011781, 1006104
+---
+
+## Environment 
+<table>
+  <tr>
+    <td>Service</td>
+    <td>Code (AppBuilder)</td>
+  </tr>
+  <tr>
+    <td>Feature</td>
+    <td>Run on Device</td>
+  </tr>
+  <tr>
+    <td>Mobile development type</td>
+    <td>Hybrid (Cordova app)</td>
+  </tr>
+</table>
+
+## Description
+When running an app on a device, its requests to a remote point via HTTPS do not work, even though the app works in the device simulator. Upon expecting the error arguments of the failed request, you find `statusText: error` and no error message or response text.
+
+The issue is observed on iOS devices with any build configuration and on Android devices when the app is built with **Release** configuration.
+
+## Cause
+This problem occurs when the remote server uses an untrusted SSL certificate. This could happen if the server uses a certificate form a trusted authority but does not have any intermediate certificates installed, or if the server uses a self-signed certificate. You can check if there is such a problem with your server certificate by running it through an online SSL certificate checker tool like [Symantec CryptoReport](https://cryptoreport.symantec.com/checker/).
+
+Android devices allow connections via HTTPS to a server using an untrusted certificate only when you build the app with **Debug** build configuration, to let you test your app. When the app is built with **Release** configuration, connections to such servers are blocked.
```

Comment:
What about iOS devices. Also - this is not device-related, it is web view logic. Perhaps we may state a more abstract reasoning. Debug build on Android is an exception to the general rule.